Lib/glyphsLib/builder/custom_params.py

```diff
@@ -310,52 +310,53 @@ def register(handler):
     KNOWN_PARAM_HANDLERS.append(handler)
 
 
-GLYPHS_UFO_CUSTOM_PARAMS = (
-    ("hheaAscender", "openTypeHheaAscender"),
-    ("hheaDescender", "openTypeHheaDescender"),
-    ("hheaLineGap", "openTypeHheaLineGap"),
-    ("compatibleFullName", "openTypeNameCompatibleFullName"),
-    ("WWSSubfamilyName", "openTypeNameWWSSubfamilyName"),
+GLYPHS_UFO_MAPPING = (
```

Comment:
why did you merge those three lists of tuples into one? I don't particularly find a single list (with a bunch of Nones) more readable...

Reply:
The problem is not the Bones, the different lists grouped similar items that received similar treatment, which a single list obfuscates a bit. I don't mind the git diff changes.